src/effects/VST3/internal/PlugFrame.h

```diff
@@ -0,0 +1,34 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file PlugFrame.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
```

Comment:
I am pleased to see that the new files in internal/ have no dependency cycles.  I can study them in a bottom-up order to understand them.

Please write a little more commentary about the purposes of the classes.